src/pages/Search/AdvancedSearchFilters.tsx

```diff
@@ -213,6 +219,16 @@ const typeFiltersKeys: Record<string, Array<Array<ValueOf<typeof CONST.SEARCH.SY
     ],
 };
 
+function getFilterWorkspaceDisplayTitle(filters: Partial<SearchAdvancedFiltersForm>, policies: OnyxCollection<Policy>) {
+    const workspaceFilter = filters[CONST.SEARCH.SYNTAX_FILTER_KEYS.POLICY_ID];
+
+    if (!policies) {
+        return '';
+    }
```

Comment:
this is temporary right? eventually we need to fallback to something so that we display it for user. Something like `"unknown"` or simply `policyID`?